### DISCUSSION TITLE
Can Mirth connect to pgbouncer -> postgresql?

### DISCUSSION THREAD

**Discussion by rjcampion3** [nextgenhealthcare/connect#5338] [Q&A]
I tried connecting to a postgresql database through pgbouncer in transaction mode and it appears that pgbouncer can't accept prepared statements. Is there a way to disable prepared statements in Mirth? 

Mirth side
` | ### Error updating database.  Cause: org.postgresql.util.PSQLException: ERROR: prepared statement "S_16" already exists`

Postgresql side:
```patroni3_1  | 2022-08-05 21:31:17.103 UTC [1774] ERROR:  prepared statement "S_3" already exists
patroni3_1  | 2022-08-05 21:31:17.103 UTC [1774] STATEMENT:  COMMIT
patroni3_1  | 2022-08-05 21:31:17.146 UTC [1774] ERROR:  prepared statement "S_3" does not exist
patroni3_1  | 2022-08-05 21:31:17.150 UTC [1774] ERROR:  prepared statement "S_4" already exists
patroni3_1  | 2022-08-05 21:31:17.150 UTC [1774] STATEMENT:  ROLLBACK
patroni3_1  | 2022-08-05 21:31:17.162 UTC [1774] ERROR:  prepared statement "S_5" already exists
patroni3_1  | 2022-08-05 21:31:17.162 UTC [1774] STATEMENT:  COMMIT
patroni3_1  | 2022-08-05 21:31:17.165 UTC [1774] ERROR:  prepared statement "S_4" does not exist
```

**Comment by jonbartels** [comment]
https://www.pgbouncer.org/faq.html - Sounds like you might need a different argument to JDBC or to use a different pooling strategy.

I am not an expert in pgBouncer but I recall other users having used it with MC. Anything on https://forums.mirthproject.io/? Anything in GH discussion history?
